posthog/models/team/team.py

```diff
@@ -130,9 +130,9 @@ class Team(UUIDClassicModel):
     anonymize_ips: models.BooleanField = models.BooleanField(default=False)
     completed_snippet_onboarding: models.BooleanField = models.BooleanField(default=False)
     ingested_event: models.BooleanField = models.BooleanField(default=False)
-    session_recording_opt_in: models.BooleanField = models.BooleanField(default=False)
-    capture_console_log_opt_in: models.BooleanField = models.BooleanField(null=True, blank=True)
-    capture_performance_opt_in: models.BooleanField = models.BooleanField(null=True, blank=True)
+    session_recording_opt_in: models.BooleanField = models.BooleanField(default=True)
```

Comment:
@alexkim205 do we actually want to default this to `True` for a new team? I thought the idea was to only turn it on for the team _after_ they've seen the ingestion screen and explicitly hit `Continue` with the option set to On.

Reply:
Hmm, I was making the assumption here that for all new projects, the user would have gone through the ingestion screen. If we change the default here to `False` we'd have to have extra logic either in the FE or BE that turns these settings on when the onboarding flow is seen—which would be the most accurate way to implement it. Is there the case where a user can manually skip the onboarding flow entirely?

Reply:
I think avoiding the migration and defaulting to `False` is the better way to go. That way if someone adds the snippet to their app but gets distracted and never finishes onboarding then we aren't suddenly storing a bunch of session recording data for them. OR say they add the snippet to their site, get distracted, come back a day later, say "no I don't want session recordings", finish onboarding and then go to find that there are actually session recordings from the prior day. I can imagine some side-eye happening there.

I think those two situations are unlikely but I've already handled it on the front-end and we can avoid a migration so I say we do that.

Reply:
That's ideal for any caching logic too, fwiw, as regular updates are handled perfectly well in the cache